components/brave_wallet_ui/components/shared/create-placeholder-icon/index.tsx

```diff
@@ -123,19 +110,12 @@ export function withPlaceholderIcon<
       }
     }, [needsPlaceholder, asset?.contractAddress, asset?.name])
 
-    const remoteImage = React.useMemo(() => {
-      if (isRemoteURL) {
-        return isStorybook ? ipfsUrl || '' : `chrome://image?${ipfsUrl}`
-      }
-      return ''
-    }, [isRemoteURL, ipfsUrl])
-
     // render
     if (!asset) {
       return null
     }
 
-    const icon = nativeAssetLogo || (isRemoteURL ? remoteImage : asset?.logo)
+    const icon = nativeAssetLogo || (asset?.logo)
```

Comment:
nit: unneeded parens
```suggestion
    const icon = nativeAssetLogo || asset?.logo
```